src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -295,4 +295,32 @@ public enum StreamMetadataKey
         /// </summary>
         Year
     }
+
+    /// <summary>
+    /// Enumeration of audio extract option.
+    /// </summary>
+    /// <seealso cref="Player.EnableExportingAudioData"/>
+    /// <since_tizen> 6 </since_tizen>
+    public enum PlayerAudioExtractOption
+    {
+        /// <summary>
+        /// Sync with the playback clock and multichannel audio stream
+        /// </summary>
+        Default = 0x00,
+
+        /// <summary>
+        /// No sync with the playback clock
+        /// </summary>
+        NoSyncWithClock = 0x01,
+
+        /// <summary>
+        /// Splits one interleaved multichannel audio stream into many mono audio streams
+        /// </summary>
+        Deinterleave = 0x02,
+
+        /// <summary>
+        /// No sync with clock and splits into mono streams
```

Comment:
What does the sentence mean?
// Doesn't sync with clock, but splits multi-channel audio stream into several mono streams.
or
// Doesn't sync with clock and doesn't split multi-channel audio stream into several mono audio streams.

Reply:
https://github.com/Samsung/TizenFX/pull/955
i've updated it